cdap-ui/app/cdap/components/FieldLevelLineage/v2/Context/FllContextHelper.ts

```diff
@@ -268,12 +269,20 @@ export function getTimeRangeFromUrl() {
   return { selection, range: TIME_OPTIONS_MAP[selection] };
 }
 
+// Currently we are just measuring the size of the response. This will be updated once
+// there is proper pagination for the FLL response from backend. This limit is approximately
+// 1500 fields going into 1500 fields
+function isFLLResponseTooLarge(response) {
+  const CHARACTER_LIMIT = 50000;
```

Comment:
Should this be defined at the top of the file?

Reply:
the constant is not really applicable anywhere else